src/App.js

```diff
@@ -15,6 +15,7 @@ function App() {
         <Route path="/" element={<LoginPage />} />
         <Route path="/signup" element={<RegistrationPage />} />
         <Route element={<AuthenticateUser />}>
+          <Route path="/addCohort" element={<AddCohort />} />
```

Comment:
this path might be better as `add-cohort` or maybe `cohort/add`. You want to avoid using upper case letters in the url

Reply:
I've removed this path as it's no longer necessary.